docs/feature.md

```diff
@@ -0,0 +1,74 @@
+# Featured People
+## _Organizing Yaml Files_
+
+Data collected through the nominate form will be extracted and then inputted into the Currently Featured and All Featured Yaml files.
+
+The Yaml template file located in this folder represents a template for a currently featured file. Make a copy of this file and fill out the fields according to the format specified in the **Currently Featured Files** section. This copied file should then be placed into the data folder which is located in the `humans/humans-proj/src` folder. The data in the template should also be copied and placed in the all featured Yaml file with the same format, alongside the other entries. The layout is shown below.
+
+The new entry should be under the Humans header.
+
+```
+Humans: 
+  - id: 
+    date: 
+    first_name: 
+    last_name: 
+    position: 
+    photo: 
+    photo_alt: 
+    description: >
+    description2: 
+    socials: 
+      instagram:
+      facebook:
+      twitter:
+      internet:
+      <other>:
+
+  - id: Date_NameLastName
+    date: Date
+    first_name: Name
+    last_name: Lastname
+    position: Position
+    photo: Photo.png
+    photo_alt: Name Lastname
+    description: >
+      Description
+    description2: 
+      Description2
+    socials: 
+      instagram: "https://www.instagram.com/"
+      facebook: "https://www.facebook.com/"
+      twitter: "https://twitter.com/home"
+      internet: "https://www.google.ca/"
+```
+
+## Currently Featured Files
+
+These files are stored in the `humans/humans-proj/src` folder.
+Every file should have a unique ID associated with it which will represent the file name.
+The ID will be comprised of the date, the person's first name and their last name. 
+The ID is represented as `Date_FirstNameLastName` in the Yaml file and the file name will be `DateFirstNameLastName.yaml`
```

Comment:
what is the format of the date? is it just DD/MM/YYYY or is it in words so "April2nd2022". Specify date format.

Reply:
nvm see the format below